doc/source/install/compile.rst

```diff
@@ -108,11 +108,14 @@ To compile |PXC| from source code:
 
    .. code-block:: bash
 
-      $ git submodule init wsrep/src && git submodule update wsrep/src
+      $ git submodule init wsrep-lib && git submodule update wsrep-lib
```

Comment:
It's more reliable to do this one command to capture all the submodules needed.

`git submodule update --init --recursive`

Reply:
Updated.